cookbook/core/scheduled_workflows/lp_schedules.py

```diff
@@ -42,7 +40,9 @@ def date_formatter_wf(kickoff_time: datetime):
     workflow=date_formatter_wf,
     schedule=CronSchedule(
         # Note that kickoff_time_input_arg matches the workflow input we defined above: kickoff_time
-        cron_expression="0 10 * * ? *",
+        # Native scheduler doesn't support `AWS syntax <http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#CronExpressions>`_.
```

Comment:
lets add this in the upper description and make it into a

..  note:

Reply:
Done